Agreements/Internal/IoM_Wal.txt

```diff
@@ -2,10 +2,10 @@ COPX:*:*:IBRAR:EGCC:*:Isle of Man:Wallasey:*:17000:IBRAR
 COPX:*:*:PENIL:EGCC:*:Isle of Man:Wallasey:*:17000:PENIL
 COPX:*:*:MALUD:EGCC:*:Isle of Man:Wallasey:*:17000:MALUD
 
-COPX:*:*:PENIL:EGGP:*:Isle of Man:Wallasey:*:13000:|PENIL
-COPX:*:*:PENIL:EGNR:*:Isle of Man:Wallasey:*:13000:|PENIL
-COPX:*:*:MALUD:EGGP:*:Isle of Man:Wallasey:*:13000:|MALUD
-COPX:*:*:MALUD:EGNR:*:Isle of Man:Wallasey:*:13000:|MALUD
+COPX:*:*:PENIL:EGGP:*:Isle of Man:Wallasey:*:13000:PENIL
+COPX:*:*:PENIL:EGNR:*:Isle of Man:Wallasey:*:13000:PENIL
+COPX:*:*:MALUD:EGGP:*:Isle of Man:Wallasey:*:13000:MALUD
+COPX:*:*:MALUD:EGNR:*:Isle of Man:Wallasey:*:13000:MALUD
```

Comment:
Source? My source still says descending agreement

Reply:
PC MATS - correct as of March 2020

Reply:
(Would need to update in the temp instruction I sent you if this were to be merged before next week's event! 😄)

Reply:
As I said, my version of this is (still) descending, also current for 23 April; I won't know what the level by would add -> its neither airspace or STAR point related. Clearly an error in one of the documents, but not clear which is correct in my view.